<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userinterface</class>
 <widget class="QWidget" name="userinterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:qradialgradient(spread:reflect, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(27, 58, 249, 255), stop:1 rgba(255, 255, 255, 255))</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">qproperty-alignment:'AlignCenter';
color:black;
font: 900 22pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string> MANAGEROOM X V2.0.2</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupinscription">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>951</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>Veuillez renseigner vos infomations pour que seul vous ou des personnnes que vous avez autorisé puissenr accéder à votre compte</string>
   </property>
   <widget class="QTextEdit" name="username">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="mail">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="password">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="role">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>721</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <item>
     <property name="text">
      <string>administrateur</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Utilisateur(client)</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">qproperty-alignment:'AlignCenter';</string>
    </property>
    <property name="text">
     <string>Nom d'utilisateur:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">qproperty-alignment:'AlignCenter';</string>
    </property>
    <property name="text">
     <string>adresse email:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">qproperty-alignment:'AlignCenter';</string>
    </property>
    <property name="text">
     <string>Mot de passe:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">qproperty-alignment:'AlignCenter';</string>
    </property>
    <property name="text">
     <string>date de naissance</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">qproperty-alignment:'AlignCenter';</string>
    </property>
    <property name="text">
     <string>Role dans l'entreprise</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="birthday">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>721</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="inscription">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>580</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:red;
color:white;</string>
   </property>
   <property name="text">
    <string>S'inscrire</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connexion">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>580</y>
     <width>151</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:red;
color:white;</string>
   </property>
   <property name="text">
    <string>Se connecter</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupconnexion">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Veuillez renseigner vos informations de connexion</string>
   </property>
   <widget class="QTextEdit" name="mail2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>811</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="password2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>811</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>email de connexion:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">qproperty-alignment:'AlignCenter';</string>
    </property>
    <property name="text">
     <string>Mot de passe:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connexion2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:red;
color:white;</string>
    </property>
    <property name="text">
     <string>Se connecter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inscription2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>601</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:red;
color:white;</string>
    </property>
    <property name="text">
     <string>Créer un compte</string>
    </property>
   </widget>
  </widget>
  <widget class="QProgressBar" name="loading">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>580</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
